lib/semantics/resolve-labels.cc

```diff
@@ -637,21 +656,21 @@ class ParseTreeAnalyzer {
   void addLabelReferenceFromDoStmt(parser::Label label) {
     CheckLabelInRange(label);
     programUnits_.back().doStmtSources.emplace_back(
-        std::tuple<parser::Label, ScopeProxy, parser::CharBlock>{
+        std::tuple<parser::Label, ProxyForScope, parser::CharBlock>{
             label, currentScope_, currentPosition_});
```

Comment:
Because you're using emplace_back, I don't think you need the type and braces. Just:
`emplace_back(label, currentScope_, currentPosition_)`

Reply:
part of the TuplePODing changes